Cargo.toml

```diff
@@ -49,6 +49,9 @@ edition = "2021"
 license = "MIT"
 repository = "https://github.com/denoland/deno"
 
+[patch.crates-io]
+deno_config = { git = "https://github.com/denoland/deno_config.git", rev = "22cd23bacd29f115c3f54a078acb51b72e73e46e" }
+
```

Comment:
Merge https://github.com/denoland/deno_config/pull/152 and remove this before landing.

This is running on a temp branch for now https://github.com/denoland/deno_config/commits/22cd23bacd29f115c3f54a078acb51b72e73e46e.